### DISCUSSION TITLE
Regarding layers in osm

### DISCUSSION THREAD

**Discussion by jesussmile** [liodali/osm_flutter#398] [Show and tell]
Hi liodali,
Check this repo I made that uses canvas to render 30k markers https://github.com/jesussmile/markerx , I want to do sth like this for your package too.

In flutter_map I can just use a layer and build anything I want on top of it.
As everything in ur is going to native view it is difficult for us to help out 
It is a big project and eventually it will be too much for you to manage.

If I want to do sth like this to ur project, how can I do it?

**Comment by liodali** [comment]
for sure you free to do it, and it will be great that you can help us
